ui/shared/Layout.tsx

```diff
@@ -197,6 +197,12 @@ export const Layout: React.FC = ({ children }) => {
           <a href="https://twitter.com/TheGuildDev" title="Visit our Twitter">
             <Twitter />
           </a>
+          <a
+            href="https://www.youtube.com/channel/UCY5MRYIB_x1i4wPZ20aZTvQ"
+            title="Visit our YouTube channel"
```

Comment:
@danielrearden because we still don't have content, maybe it's better to point directly to the playlist section?